huaweicloud/utils/utils.go

```diff
@@ -63,6 +63,17 @@ func ExpandToStringList(v []interface{}) []string {
 	return s
 }
 
+// ExpandToIntList takes the result for an array of intgers and returns a []int
+func ExpandToIntList(v []interface{}) []int {
+	s := make([]int, 0, len(v))
+	for _, val := range v {
+		if strVal, ok := val.(int); ok {
```

Comment:
use intVal instead of strVal could be better

Reply:
done